packages/terra-docs/src/Button.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Button.module.scss';
+
+const KEY_RETURN = 13;
+const KEY_SPACE = 32;
```

Comment:
I'm not importing keycodes. Part of my goal with this package was to not add any dependencies beyond react.